spring-boot-samples/spring-boot-sample-cache/src/main/resources/geode.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:context="http://www.springframework.org/schema/context"
+       xmlns:gfe="http://www.springframework.org/schema/geode"
+       xmlns:util="http://www.springframework.org/schema/util"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:schemaLocation="
+        http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd
+        http://www.springframework.org/schema/context http://www.springframework.org/schema/context/spring-context.xsd
+        http://www.springframework.org/schema/util http://www.springframework.org/schema/util/spring-util.xsd
+        http://www.springframework.org/schema/geode http://www.springframework.org/schema/geode/spring-geode.xsd
+">
+
+  <context:property-placeholder/>
+
+  <util:properties id="gemfireProperties">
+    <prop key="name">SampleCacheUsingGeodeApplication</prop>
+    <prop key="mcast-port">0</prop>
+    <prop key="log-level">${sample.cache.gemfire.log.level:config}</prop>
+  </util:properties>
+
+  <gfe:cache properties-ref="gemfireProperties"/>
+
+  <gfe:local-region id="countries" persistent="false"/>
+
+</beans>
```

Comment:
Is that the only way to configure it?  That is weird for us to introduce an xml config in a sample...